docs/dev/table/python/usage_notes.md

```diff
@@ -0,0 +1,120 @@
+---
+title: "Usage Notes"
```

Comment:
What do you think about changing this to "Common Questions"? 
```suggestion
title: "Common Questions"
```

Reply:
Yes. Make sense.